### ISSUE TITLE
Feature request: Option to show the passcode in the clear during entry

### ISSUE THREAD

**Issue by torpax** [standardnotes/forum#259]
I frequently have a problem correctly typing the passcode on my Android.  Only after hitting enter does it say the code is incorrect.  So it would nice to have a box you could hit that would then show the passcode in the clear as you enter.  That way it would be easy to see any typos and correct them prior to hitting enter.

Lots of other applications have this option.

**Comment by mobitar** [comment]
Thanks for the feedback, will keep this in mind.

**Comment by effieeee** [comment]
Already implemented :)
